yt_dlp/extractor/nebula.py

```diff
@@ -99,6 +107,7 @@ def _build_video_info(self, episode):
             'uploader_url': f'https://nebula.tv/{channel_slug}',
             'series': channel_title,
             'creator': channel_title,
+            '_old_archive_ids': [make_archive_id(self, zype_id)] if zype_id else None,
```

Comment:
The playlist classes yield from this method, so we want to specify video extractor in the old archive id
```suggestion
            '_old_archive_ids': [make_archive_id(NebulaIE, zype_id)] if zype_id else None,
```

Reply:
actually, since this baseclass method apparently never set the `extractor`/`extractor_key` to the video extractor, this change would not be helpful

Reply:
but if we are changing the video ID, that gives a chance to correct this by adding:
```py
            'extractor_key': NebulaIE.ie_key(),
            'extractor': NebulaIE.IE_NAME,
```

Reply:
Add a comment to the `self` saying it's intentional.

Reply:
I think I am going to do this after all:
```suggestion
            '_old_archive_ids': [make_archive_id(NebulaIE, zype_id)] if zype_id else None,
```
it doesn't really matter, since before it was essentially random what it would've been recorded in the download archive as. at least the code will be consistent this way